libs/watt/src/lib/components/chip/watt-action-chip.component.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, input, output } from '@angular/core';
+
+import { WattIcon } from '../../foundations/icon/icons';
+import { WattChipComponent } from './watt-chip.component';
+import { WattIconComponent } from '../../foundations/icon/icon.component';
+
+@Component({
+  standalone: true,
+  imports: [WattChipComponent, WattIconComponent],
+  selector: 'watt-action-chip',
+  styles: [
+    `
+      :host {
+        display: block;
+      }
+
+      .disabled {
+        color: var(--watt-on-light-low-emphasis);
+      }
+    `,
+  ],
+  template: `
+    <watt-chip [disabled]="disabled()">
+      <button
+        type="button"
+        class="cdk-visually-hidden"
+        (click)="$event.stopImmediatePropagation(); onClick.emit()"
+        [disabled]="disabled()"
+      ></button>
```

Comment:
I did not think much about it, I just copied one of the other chip components. But isnt it more correct accessibility to have a button element than a div with a click event?

Reply:
Yeah, when tabbing between the elements in the "Edit organization" modal, where this component is used, indeed docus is set on the chip when it includes a `button` element, and the component is skipped when it doesn't [include a `button`].

The reason for being "suspicious" of `$event.stopImmediatePropagation()` here is because it's used to counter a side-effect caused only when [the component is] used in combination with another component. But since the `onClick` event is still emitted it's most likely okay to call `$event.stopImmediatePropagation()`. The parent component is still notified when click happens.